packages/gatsby/src/schema/build-connection-fields.js

```diff
@@ -104,7 +104,30 @@ module.exports = type => {
           groupConnections.push(groupConn)
         })
 
-        return groupConnections
+        // make tags arrays case ignored
+        const tags = groupConnections.map(e=>{
```

Comment:
Wow, didn't know `gatsby-transformer-react-docgen` exist, not even included on the website,  I'll check on it, thanks!